Dev/GitHub/prnotii.5m.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
+
+// <xbar.title>GitHub PRNotii</xbar.title>
+// <xbar.version>v1.0</xbar.version>
+// <xbar.author>aiden.ahn</xbar.author>
+// <xbar.author.github>eu81273</xbar.author.github>
+// <xbar.desc>Check Github Pull Reuqest</xbar.desc>
+// <xbar.image>https://github.com/matryer/xbar-plugins/assets/2687916/e08d6380-62f9-478b-b35b-e2222cf65791</xbar.image>
```

Comment:
Should use the actual image path here.

```suggestion
// <xbar.image>https://github-production-user-asset-6210df.s3.amazonaws.com/2687916/246659270-e08d6380-62f9-478b-b35b-e2222cf65791.png</xbar.image>
```